packages/core/src/browser/keybinding.spec.ts

```diff
@@ -331,15 +331,39 @@ describe("keys api", () => {
         expect(KeyCode.equals(parsedKeyCode, keyCode)).to.be.true;
     });
 
-    it("it should seralize a keycode properly with a + M2 + M3", () => {
+    it("it should serialize a keycode properly with a + M2 + M3", () => {
         const keyCode = KeyCode.createKeyCode({ first: Key.KEY_A, modifiers: [Modifier.M2, Modifier.M3] });
         const keyCodeString = keyCode.toString();
         expect(keyCodeString).to.be.equal("shift+alt+a");
         const parsedKeyCode = KeyCode.parse(keyCodeString);
         expect(KeyCode.equals(parsedKeyCode, keyCode)).to.be.true;
     });
 
-    it("it should seralize a keycode properly with a + M4", () => {
+    it("the order of the modifiers should not matter when parsing the key code", () => {
```

Comment:
I know it's not exactly related to this changeset, but shouldn't we usually type mocha tests like so ->
`it("should....")`? I just noticed the ones above repeat the `it should` but this one doesn't have either. What's the correct convention here?

Reply:
If we would like to follow such a convention, could you please create a follow-up task, where we define the preferred way, adjust the existing tests and document it in our [guideline](https://github.com/theia-ide/theia/wiki/Coding-Guidelines)?